### PR TITLE
[13.0][ADD] New module 'stock_picking_delivery_link'

### PR DESCRIPTION
**Pull request by sebalix**
Adds link to the delivery on all intermediate operations (pick and pack with a pick/pack/ship setup).

It aims to replace https://github.com/OCA/stock-logistics-warehouse/pull/1115 to get a useful module on its own instead of a "helper" one.

image

Ref. 2356

### CODE REVIEW

File: stock_picking_delivery_link/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    )
+    ship_carrier_id = fields.Many2one(
+        comodel_name="delivery.carrier",
+        compute="_compute_ship_picking_data",
+        string="Related carrier",
+    )
+
+    def _compute_ship_picking_data(self):
+        for picking in self:
+            ship = self._get_ship_from_chain()
```
Comment (simahawk) [inline stock_picking_delivery_link/models/stock_picking.py]: :fire: 
```suggestion
            ship = picking._get_ship_from_chain()
```

**Review by simahawk — CHANGES_REQUESTED** [review]
shouldn't stay in delivery-carrier?

File: stock_picking_delivery_link/models/stock_picking.py (hunk 1 above)
Comment (sebalix) [reply stock_picking_delivery_link/models/stock_picking.py]: fixed

**Comment by sebalix** [comment]
Not sure, to me it's more related to the workflow of operations than the configuration of delivery carriers (https://github.com/OCA/delivery-carrier/#readme )

**Comment by pedrobaeza** [comment]
Shouldn't this go in OCA/delivery-carrier?

**Comment by sebalix** [comment]
@pedrobaeza As said I don't think but I could move it there if you prefer?

**Comment by pedrobaeza** [comment]
For me all modules depending on `delivery` has that special repo, as this one is very populated, but I don't want to impose it.

**Comment by simahawk** [comment]
@sebalix to be honest it makes sense. If I'm looking for something related to "delivery" that's the 1st place where I'd go normally.

**Comment by sebalix** [comment]
Moved to https://github.com/OCA/delivery-carrier/pull/354, closing this one.

**Pull request closed without merging**